nav2_route/src/route_server.cpp

```diff
@@ -13,6 +13,7 @@
 // limitations under the License. Reserved.
 
 #include "nav2_route/route_server.hpp"
+#include "nav2_core/planner_exceptions.hpp"
```

Comment:
Why planner exceptions?

Reply:
We talked about this while back. The breadth first search throws a lot of the same errors as the planner server. We could add a separate set for errors for bfs but it would be just be a subset of the planner server errors.

Reply:
Ah ok, but we're returning in all actions the route-specific failure codes, right?

Reply:
Yeah, 

the error code we use have been added to the action message.